cmd/shortener/main.go

```diff
@@ -1,3 +1,33 @@
 package main
 
-func main() {}
+import (
+	"net/http"
+
+	"mishin-shortener/internal/app/config"
+	"mishin-shortener/internal/app/handlers"
+	"mishin-shortener/internal/app/storage"
+
+	"fmt"
+	"github.com/go-chi/chi/v5"
+)
```

Comment:
Было бы здорово отсортировать импорты. Автоматическую сортировку при сохранении можно включить в VSCode.

```suggestion
	"fmt"
	"net/http"

	"mishin-shortener/internal/app/config"
	"mishin-shortener/internal/app/handlers"
	"mishin-shortener/internal/app/storage"

	"github.com/go-chi/chi/v5"
)
```

Reply:
Что за чудо этот VScode) Последние 10 лет использовал atom.
Настроил, исправлено